### ISSUE TITLE
Looking to add new records [Support]

### ISSUE THREAD

**Issue by TobezDev** [is-a-dev/register#14003]
### Your subdomain
tobez.is-a.dev

### The issue
I'm looking to add a new TXT record to my subdomain.
I would do this with a PR however I am still working on #13982 and:
* I don't want to mix up this new domain update with adding the formatting;
* I cannot open more than one PR at a time from the same repository.

If it's possible to be done manually, please let me know and I'll provide the record content to be added. Otherwise, I'll create a new PR after #13982 is complete.

Many thanks.

**Comment by TobezDev** [comment]
I feel like a moron that I've forgotten this. Thanks :/
